MIGRATING.md

```diff
@@ -0,0 +1,143 @@
+# Migrate from the SignalFx Tracing Library for NodeJS
+
+The Splunk Distribution of OpenTelemetry for NodeJS replaces the SignalFx Tracing
+Library for NodeJS. If you’re using the SignalFx Tracing Library, migrate to
+the Splunk Distribution of OpenTelemetry for NodeJS to use OpenTelemetry’s
+instrumentation to send traces to Splunk APM. The Splunk Distribution of
+OpenTelemetry for NodeJS uses OpenTelemetry to instrument applications, which is
+an open-source API to gather telemetry data, and has a smaller footprint.
+
+Because the SignalFx Tracing Library for NodeJS uses OpenTracing and the Splunk Distribution
+of OpenTelemetry for NodeJS uses OpenTelemetry, the semantic
+conventions for span tag names change when you migrate. For more information,
+see [Migrate from OpenTracing to OpenTelemetry](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html#apm-opentelemetry-migration).
+
+<a name="known-limitations"></a>
+## Known limitations as compared to SignalFx Tracing Library
+
+- Lowest supported version of NodeJS is `v8.5`, [see more information here](https://github.com/open-telemetry/opentelemetry-js#node-support)
+- No auto-instrumentation for:
+  - `AdonisJS`
+  - `amqp10`
+  - `mongodb-core` ([because it's deprecated](https://github.com/mongodb-js/mongodb-core))
+  - `sails`
+- Limited instrumentation for:
+  - `nest` - only manual insturmentation helpers, provided by community
+- other notes on instrumentation:
+  - `express` instrumentation requires `http`/`https` instrumentation
+  - `bluebird` - context propagation only (via `async_hooks`)
+  - `q` - context propagation only (via `async_hooks`)
+  - `when` - context propagation only (via `async_hooks`)
+  - `socket.io` - provided by community (<https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/instrumentation-socket.io>)
+
+## Requirements
+
+This Splunk Distribution of OpenTelemetry requires Node.js 8.5 or later.
+If you're still using an earlier version of Node.js, continue using the SignalFx Tracing Library for Node.js.
+
+## Equivalent configurations
+
+### Changes to defaults
+
+- default flush interval is now 30s instead of 2s (how frequently captured telemetry data is sent to the backend)
+
+### Instrumented libraries
+
+With the exception of [explicitly listed limitations](#known-limitations) we aim to support all libraries supported by
+signalfx-nodejs-tracing. To find an equivalent auto-instrumentation open <https://opentelemetry.io/registry/> and for
+each instrumentation
+[from `signalfx-nodejs-tracing`'s README](https://github.com/signalfx/signalfx-nodejs-tracing/#requirements-and-supported-software)
+search by the name of the library in the registry.
+
+For example, if you'd like to migrate instrumentation for `mysql`, go to
```

Comment:
```suggestion
For example, if you'd like to migrate instrumentation for MySQL, go to
```

Reply:
Well, actually, this is how the package is called, see <https://www.npmjs.com/package/mysql>. Maybe I should convert it into a link and then it'll be clearer?